### ISSUE TITLE
2021 Questionnaire Setup Ontario

### ISSUE THREAD

**Issue by rennera** [RADB/Offord-EDI-2020#116]
Patricia has requested the questionnaire for Ontario to be setup for a small run.

**Comment by rennera** [comment]
First Step is to include the province in the "year settings"
![image](https://user-images.githubusercontent.com/3689865/121604720-07df4d00-ca19-11eb-8a74-9317dffea780.png)

**Comment by rennera** [comment]
Second Step include the teacher profile questionnaire for the province.
![image](https://user-images.githubusercontent.com/3689865/121604783-26dddf00-ca19-11eb-8607-ad56afa6d2f5.png)

**Comment by rennera** [comment]
Third Step -- Add the site, school and teacher

**Comment by rennera** [comment]
Update the config for teacher profile to include Ontario
`UPDATE [EDI].[Questionnaires.Configuration] SET [EntityName] ='Questionnaires.Data.TeacherProfile',[EntityField] =[VariableName],NorthwestTerritories=1, novascotia = 1, Ontario=1,YearID =1 WHERE QuestionnaireID=1`

**Comment by rennera** [comment]
Use 2020 EDI Tables excel file to configure the questionnaire and generate the Insert statements.
`




<meta name="ProgId" content="Excel.Sheet">
<meta name="Generator" content="Microsoft Excel 15">
<link id="Main-File" rel="Main-File" href="file:///C:/Users/andre/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel="File-List" href="file:///C:/Users/andre/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>





INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,9,1,1,0,'Title',0,Null,Null,Null,'Null','Child Demographics','Données   personnelles de l’enfant','NULL',0,'NULL',Null,10,1,1,'NULL'   ,0,'','',1,0,'null','','')
--
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,11,1,1,0,'Status',1,Null,Null,Null,'Null','','','NULL',0,'NULL',NULL,20,1,1,'NULL'   ,0,'','',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,3,1,0,0,'Status',0,Null,Null,Null,'Null','Student Status','Situation   de l’élève','StudentStatus',1,'LookupSets',26,30,1,1,'NULL'   ,1,'&lt;p&gt;&lt;b&gt;14. Student Status&lt;/b&gt;&lt;/p&gt;&lt;p&gt;·   Student must be currently in your class for you to complete the EDI. If the   child is currently in your class but has been there for less than one month,   do not complete the rest of the form. This applies to children who have   entered the class at some time other than the beginning of the year or who   have been off sick or travelling.&lt;/p&gt;&lt;p&gt;· If the child has moved   out of this class/school, do not complete the rest of the   questionnaire.&lt;/p&gt;&lt;p&gt;· If parents/guardians request that you do   not assess their child, mark &lt;b&gt;other &lt;/b&gt;and do not fill in the   rest of the questionnaire.&lt;/p&gt;','&lt;p&gt;&lt;b&gt;14. Situation de   l’élève&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Si l’enfant est dans votre classe   depuis&lt;b&gt; moins d’un mois&lt;/b&gt;, ne remplissez pas le restedu   formulaire. Cela s’applique aux enfants qui se sont joints à la classe après   la rentrée scolaire, qui ont été malades ou partis en   voyage.&lt;/p&gt;&lt;p&gt;· De même, si l’enfant n’est plus dans cette   classe/école ne remplissez pas le reste du formulaire.&lt;/p&gt;&lt;p&gt;· Si   les parents/tuteurs ne veulent pas faire évaluer leur enfant, veuillez   sélectionner &lt;b&gt;Autre &lt;/b&gt;et ne remplissez pas le reste du   questionnaire.&lt;/p&gt;',1,1,'&gt;1','Student must currently be in your   class to do the EDI. If the child is in your class but has been there for less   than a month, if he/she has changed classes or schools, do not complete the   rest of the form. Check for completeness and the finish/sent to   McMaster.','Si l’enfant est dans votre classe depuis moins d’un mois, s’il   n’est plus dans votre classe ou s’il a quitté l’école, ne complétez pas le   reste du formulaire. Vérifiez l’état d’achèvement du questionnaire et faites   le parvenir à McMaster.')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,11,1,1,0,'Demographics',1,Null,Null,Null,'Null','','','NULL',0,'NULL',NULL,40,1,1,'NULL'   ,0,'','',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,3,2,0,0,'Demographics',0,Null,Null,Null,'Null','Class   Assignment:','Niveau   d''inscription','ClassAssignment',1,'LookupSets',24,50,1,1,'NULL'   ,1,'&lt;p&gt;&lt;b&gt;1. Class Assignment&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· This   question is usually pre-filled. You do not have to enter anything in this   field. &lt;/p&gt;&lt;p&gt;· In the event you do have to answer this question,   please answer as follows:&lt;/p&gt;&lt;p&gt;o Junior Kindergarten: a student   turning 4 years by December 31 of the academic year&lt;/p&gt;&lt;p&gt;o   Senior Kindergarten: Refers to a child in a class 1 year before starting   Grade 1 or a student turning 5 years by December 31 of the academic   year&lt;/p&gt;','&lt;p&gt;&lt;b&gt;1. Niveau   d’inscription&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· La réponse à cette question est   habituellement remplie de façon automatique, donc dans la plupart des cas,   vous n’aurez pas à répondre à cette question.&lt;/p&gt;&lt;p&gt;· Au cas où   vous devez répondre à cette question, veuillez utiliser les désignations   suivantes:&lt;/p&gt;&lt;p&gt;o Pré-maternelle (un élève qui aura 4 ans par le   31 décembre de l''année scolaire en question).&lt;/p&gt;&lt;p&gt;o Maternelle   : correspond à un enfant qui fréquente une classe 1 an avant la   1&lt;sup&gt;ère&lt;/sup&gt; année. &lt;/p&gt;',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,5,1,0,0,'Demographics',0,Null,Null,Null,'Null','Child''s Date of   Birth:','Date de naissance de l''enfant','Dob',0,'NULL',Null,60,1,1,'NULL'   ,1,'&lt;p&gt;&lt;b&gt;2. Child’s Date of Birth&lt;/b&gt;&lt;/p&gt; &lt;p&gt;·   Please complete only if the date of birth is INCORRECT.&lt;/p&gt; &lt;p&gt;·   If this information is missing or incorrect be sure to fill in the child’s   actual date of birth.&lt;/p&gt;','&lt;p&gt;&lt;b&gt;2. Date de   naissance&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Veuillez compléter seulement si la   date de naissance est INEXACTE.&lt;/p&gt;&lt;p&gt;· Si cette information est   manquante ou inexacte, veuillez inscrire la bonne date de naissance de   l’enfant.&lt;/p&gt;',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,3,1,0,0,'Demographics',0,Null,Null,Null,'Null','Sex','Sexe','GenderId',1,'LookupSets',10,90,1,1,'NULL'   ,1,'&lt;p&gt;&lt;b&gt;3. Child’s Sex&lt;/b&gt;&lt;/p&gt; &lt;p&gt;· Please   complete only if the child’s gender is   INCORRECT.&lt;/p&gt;','&lt;p&gt;&lt;b&gt;3. Sexe de   l’enfant&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Veuillez compléter seulement si le   sexe de l’enfant est INEXACT.&lt;/p&gt;',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,8,1,0,0,'Demographics',0,Null,Null,Null,'Null','Postal Code','Code   postal','PostalCode',0,'NULL',Null,100,1,1,'NULL' ,1,'&lt;p&gt;&lt;b&gt;4.   Child’s Postal Code&lt;/b&gt;&lt;/p&gt; &lt;p&gt;· Please complete only if   the child’s postal code is INCORRECT.&lt;/p&gt;','&lt;p&gt;&lt;b&gt;4. Code   postal de l’enfant&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Veuillez compléter seulement   si le code postal de l''enfant est INEXACT.&lt;/p&gt;',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,11,1,1,0,'ClassType',1,Null,Null,Null,'Null','','','NULL',0,'NULL',NULL,110,1,1,'NULL'   ,0,'','',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,3,1,0,0,'ClassType',0,Null,Null,Null,'Null','Class Type','Type de   classe (Voir Guide)','ClassType',1,'LookupSets',25,120,1,1,'NULL'   ,1,'&lt;p&gt;&lt;b&gt;5. Class Type&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Please   select the type of class the child is in (e.g. kindergarten,   kindergarten/grade 1, etc.)&lt;/p&gt;','&lt;p&gt;&lt;b&gt;5. Type de   classe&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Les classes peuvent se composer   uniquement d’enfants de maternelle ou de diverses combinaisons d’enfants de   maternelle et de 1ère année. Veuillez indiquer la catégorie la plus   pertinente.&lt;/p&gt;&lt;p&gt;· Pour ce qui est des options ne figurant pas   dans la liste, p. ex. Maternelle/1/2, veuillez indiquer la catégorie qui   correspond le plus étroitement (c.-à-d. Maternelle/1, dans ce   cas).&lt;/p&gt;',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,3,1,0,0,'ClassType',0,Null,Null,Null,'Null','Does the child have   exceptional/Special Needs?:','Besoins particuliers   connus','SpecialNeeds',1,'LookupSets',1,130,1,1,'NULL'   ,1,'&lt;p&gt;&lt;b&gt;1. Identified Special   Needs&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· This question must &lt;b&gt;not&lt;/b&gt;   be left blank, otherwise these children will be omitted from the   analysis.&lt;/p&gt;&lt;p&gt;· Please use the general guidelines provided   below: &lt;/p&gt;&lt;p&gt;&lt;br /&gt;&lt;/p&gt;&lt;table   border="1" cellspacing="0" cellpadding="0" &gt;   &lt;tbody&gt;&lt;tr&gt;&lt;td valign="top"   &gt;&lt;p&gt;&lt;b&gt;Yes&lt;/b&gt;&lt;/p&gt;&lt;/td&gt;&lt;td   valign="top"   &gt;&lt;p&gt;&lt;b&gt;No&lt;/b&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;&lt;p&gt;Child   has &lt;b&gt;already&lt;/b&gt; been identified as needing special assistance   due to chronic medical, physical, or mental disabling conditions, e.g.,   autism spectrum disorder, Down syndrome&lt;/p&gt;&lt;p&gt;Child requires   special assistance in the classroom&lt;/p&gt;&lt;/td&gt;&lt;td&gt;&lt;p&gt;*   * Gifted or talented * *&lt;/p&gt;&lt;p&gt;Please mark their special talents   in Section B, questions 34-39 – these are &lt;b&gt;NOT&lt;/b&gt; special   needs&lt;/p&gt;&lt;p&gt;If you only suspect that the child may be suffering   from a disabling condition, or the condition is not severe enough for the   child to be classified as “special needs.”&lt;/p&gt;&lt;p&gt;(Please indicate   the problem in Section D of the   questionnaire.)&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/tbody&gt;&lt;/table&gt;','&lt;p&gt;&lt;b&gt;7.   Besoins particuliers connus&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Cette question   &lt;b&gt;doit absolument &lt;/b&gt;être remplie, faute de quoi l’enfant ne   sera pas retenu pour les fins d’analyses.&lt;/p&gt;&lt;p&gt;· Veuillez vous   servir des lignes directrices générales fournies ci-dessous:   &lt;/p&gt;&lt;table border="1" cellspacing="0"   cellpadding="0" &gt;   &lt;tbody&gt;&lt;tr&gt;&lt;td&gt;&lt;p&gt;&lt;b&gt;Oui&lt;/b&gt;&lt;/p&gt;&lt;/td&gt;&lt;td&gt;&lt;p&gt;&lt;b&gt;Non&lt;/b&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;&lt;p&gt;Un   enfant qui a &lt;b&gt;déjà&lt;/b&gt; été identifié comme ayant besoin d’une   aide spéciale en raison d’affections incapacitantes chroniques médicales,   physiques ou mentales (par exemple, autisme, syndrome d’alcoolisme fœtal,   syndrome de Down)&lt;/p&gt;&lt;p&gt;L''enfant a besoin d''une aide spéciale   dans la classe.&lt;/p&gt;&lt;/td&gt;&lt;td&gt;&lt;p&gt;&lt;b&gt;**   Doué&lt;/b&gt; ou &lt;b&gt;talentueux&lt;/b&gt; **&lt;/p&gt;&lt;p&gt;Veuillez   plutôt inscrire leurs talents spéciaux à la section&lt;b&gt; B&lt;/b&gt;,   &lt;b&gt;questions&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;34 à   39&lt;/b&gt;&lt;/p&gt;&lt;p&gt;Si vous soupçonnez uniquement que l’enfant a   une affection incapacitante ou que l’affection n’est pas suffisamment grave   pour qu’on le classifie comme ayant des&lt;/p&gt;&lt;p&gt;&lt;b&gt;besoins   particuliers &lt;/b&gt;:&lt;/p&gt;&lt;p&gt;Veuillez indiquer le problème à la   section &lt;b&gt;D&lt;/b&gt; du   questionnaire&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/tbody&gt;&lt;/table&gt;',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,10,1,1,0,'ClassType',0,Null,Null,Null,'Null','Is the child (Check   all that apply)','Est-ce que l’enfant a des besoins particuliers?   &nbsp;(cochez toutes les cases qui   conviennent)','',0,'NULL',Null,200,1,1,'NULL' ,1,'&lt;p&gt;&lt;b&gt;13.   Child’s language(s)&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Refers to the language(s) a   child learned during his/her development and still can understand (and/or   speak).&lt;/p&gt;&lt;p&gt;· Please select all that apply from the list   provided.&lt;/p&gt;&lt;p&gt;· If the child’s language is not listed, please   select &lt;b&gt;Other&lt;/b&gt; and select the language from the dropdown   list.&lt;/p&gt;','&lt;p&gt;&lt;b&gt;13. Langue(s) maternelle(s) de   l’enfant&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· La/les langue(s) font référence aux   langues qu’un enfant a apprises au cours de son développement et qu’il peut   encore comprendre (et/ou parler).&lt;/p&gt;&lt;p&gt;· Veuillez sélectionner   toutes les langues qui conviennent de la liste fournie.&lt;/p&gt;&lt;p&gt;·   Si la langue de l’enfant ne figure pas sur la liste, veuillez   sélectionner&lt;b&gt; Autre&lt;/b&gt; et veuillez sélectionner la langue   appropriée de la liste déroulante. &lt;/p&gt;',0,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,6,1,0,0,'ClassType',0,Null,Null,Null,'Null','Identified as   exceptional by an IPRC','fut identifié comme éprouvant des difficultés par un   CIPR','IPRC',0,'NULL',Null,210,1,1,'NULL' ,0,'','',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,6,1,0,0,'ClassType',0,Null,Null,Null,'Null','Have received/in the   process of receiving an Individual Education Plan (IEP)','a reçu ou reçoit un   plan d''enseignement individualisé (PEI)','IEP',0,'NULL',Null,220,1,1,'NULL'   ,0,'','',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,6,1,0,0,'ClassType',0,Null,Null,Null,'Null','Receiving special   educational programs or services','reçoit des programmes ou des services   d''éducation de l''enfance en   difficulté','SpecialEducation',0,'NULL',Null,230,1,1,'NULL'   ,0,'','',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,6,1,0,0,'ClassType',0,Null,Null,Null,'Null','Identified as   gifted','fut identifié comme étant   doué','Gifted',0,'NULL',Null,240,1,1,'NULL' ,0,'','',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,3,1,0,0,'ClassType',0,Null,Null,Null,'Null','Child''s language   status','Statut de la langue de   l’enfant','EFSL',1,'LookupSets',1,140,1,1,'NULL' ,1,'&lt;p&gt;&lt;b&gt;8.   Child Considered ESL/FSL&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· In most school boards,   children are identified as having the ESL or FSL (English or French as a   Second Language) status if they are not fluent enough in the language of   instruction to easily follow the classroom educational activities (See table   below). &lt;/p&gt;&lt;table border="1" cellspacing="0"   cellpadding="0" &gt; &lt;tbody&gt;&lt;tr&gt;&lt;td   valign="top"   &gt;&lt;p&gt;&lt;b&gt;ESL&lt;/b&gt;&lt;/p&gt;&lt;/td&gt;&lt;td   valign="top"   &gt;&lt;p&gt;&lt;b&gt;FSL&lt;/b&gt;&lt;/p&gt;&lt;/td&gt;&lt;td   valign="top" &gt;&lt;p&gt;&lt;b&gt;No&lt;/b&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;&lt;p&gt;&lt;b&gt;Anglophone   schools:&lt;/b&gt;&lt;/p&gt;&lt;p&gt;Child for whom English is &lt;b&gt;not   &lt;/b&gt;their first language &lt;b&gt;AND&lt;/b&gt; who needs additional   instruction in   English&lt;/p&gt;&lt;/td&gt;&lt;td&gt;&lt;p&gt;&lt;b&gt;Francophone   schools:&lt;/b&gt;&lt;/p&gt;&lt;p&gt;Child for whom French is &lt;b&gt;not   &lt;/b&gt;their first language &lt;b&gt;AND&lt;/b&gt; who needs additional   instruction in French &lt;/p&gt;&lt;/td&gt;&lt;td&gt;&lt;p&gt;Child for whom   the language of instructions is the first language&lt;/p&gt;&lt;p&gt;Child   who is able to speak another language apart from the language of instruction   but is fluent in the language of instruction&lt;/p&gt;&lt;p&gt;Child whose   first language (developmentally) is &lt;b&gt;not &lt;/b&gt;the language of   instruction but is fluent in the language of   instruction&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/tbody&gt;&lt;/table&gt;','&lt;p&gt;&lt;b&gt;8.   Le français/l’anglais est la langue seconde de   l’enfant&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Dans la plupart des conseils   scolaires, les enfants sont identifiés comme ayant le français ou l’anglais   comme langue seconde s’ils ne parlent pas couramment la langue d’instruction   et ne peuvent pas suivre les activités éducationnelles en salle de classe   (voir le tableau ci-dessous).&lt;/p&gt;&lt;table border="1"   cellspacing="0" cellpadding="0" &gt;   &lt;tbody&gt;&lt;tr&gt;&lt;td valign="top"   &gt;&lt;p&gt;&lt;b&gt;ALS&lt;/b&gt;&lt;/p&gt;&lt;/td&gt;&lt;td   valign="top"   &gt;&lt;p&gt;&lt;b&gt;FLS&lt;/b&gt;&lt;/p&gt;&lt;/td&gt;&lt;td   valign="top"   &gt;&lt;p&gt;&lt;b&gt;Non&lt;/b&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;&lt;p&gt;&lt;b&gt;Écoles   Anglophones :&lt;/b&gt;&lt;/p&gt;&lt;p&gt;Enfant pour qui l’anglais n’est pas   leur langue maternelle &lt;b&gt;ET&lt;/b&gt; qui nécessite des instructions   supplémentaires en anglais.&lt;/p&gt;&lt;/td&gt;&lt;td&gt;&lt;p&gt;&lt;b&gt;Écoles   francophones :&lt;/b&gt;&lt;/p&gt;&lt;p&gt;Enfant pour qui le français n’est   pas leur langue maternelle &lt;b&gt;ET&lt;/b&gt; qui nécessite des   instructions supplémentaires en   français.&lt;/p&gt;&lt;/td&gt;&lt;td&gt;&lt;p&gt;Enfant pour qui la langue   d’instruction est sa langue maternelle.&lt;/p&gt;&lt;p&gt;Enfant qui parle   couramment une langue autre que la langue d’instruction mais parle couramment   la langue d’instruction également.&lt;/p&gt;&lt;p&gt;Enfant pour qui la   langue maternelle (de développement) n’est &lt;b&gt;pas &lt;/b&gt;la langue   d’instruction mais parle couramment la langue   d’instruction.&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/tbody&gt;&lt;/table&gt;',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,3,1,0,0,'ClassType',0,Null,Null,Null,'Null','French Immersion','Classe   d''immersion en anglais','FrenchImmersion',1,'LookupSets',1,150,1,1,'NULL'   ,1,'&lt;p&gt;&lt;b&gt;9. French Immersion&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Only   for Anglophone communities. &lt;/p&gt;&lt;p&gt;· Francophone classes/schools   are not in this category.&lt;/p&gt;&lt;p&gt;· Please indicate whether the   class is French Immersion or not. &lt;/p&gt;&lt;br /&gt;&lt;img width=575   src="images/help/image010.gif"alt="Rounded Rectangle: French   Immersion is a program in which kindergarten students are introduced early to   French language through immersion, however, the main language of the schools   remains to be English. Therefore, French Immersion teachers are asked to   respond to the EDI questions based on their observations of the children''s   abilities in English.&nbsp; It would be   unfair and inaccurate to measure the children''s ability in French as for   most students this is their first exposure to the language and assessing   their abilities in French would put them at an unfair   disadvantage.&amp;#13;&amp;#10;&amp;#13;&amp;#10;If the teacher does not feel   comfortable with the question they can indicate ''I don''t   know''.&amp;#13;&amp;#10;&amp;#13;&amp;#10;Acquisition of a second language   is strongly based on the success of the child’s acquisition of their first   language and many of the skill sets are interchangeable.&nbsp; The questions on the EDI reflect   transferable skills and abilities therefore if the child has mastered these   skills in French they have most likely also achieved this in English as   well.&amp;#13;&amp;#10;&amp;#13;&amp;#10;Teachers are requested to use their   best judgment in answering EDI items regarding skills they had not an   opportunity to observe or where it is not part of the philosophy to   test/observe the ability or behaviour in   English.&amp;#13;&amp;#10;"&gt;','&lt;p&gt;&lt;b&gt;9. Classe   d’immersion en anglais&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Uniquement pour les   collectivités francophones. &lt;/p&gt;&lt;p&gt;· Les classes/écoles   anglophones ne font pas parties de cette catégorie.&lt;/p&gt;&lt;p&gt;·   Veuillez indiquer s’il s’agit d’une classe d’immersion en anglais ou non.   &lt;/p&gt;&lt;br /&gt;&lt;p&gt;Les classes d’immersion en anglais font   parties d’un programme dans lequel les élèves de la maternelle sont   introduits à un jeune âge à l’anglais. La langue principale de l’école,   cependant, demeure le français. Donc les enseignant(e)s d’immersion en   anglais devraient répondre aux questions de l’IMDPE en fonction des   observations des habiletés des enfants en français. Il serait donc injuste et   erroné de mesurer les habiletés des enfants en anglais, car la plupart   d’entre eux viennent tout juste d’être exposé à la langue. L’évaluation de   leurs habiletés en anglais leur mettrait dans une situation désavantageuse.Si   l’enseignant(e) ne se sent pas confortable de répondre à une question en   particulier, il/elle peut sélectionner « ne sais pas ».L’acquisition d’une   langue seconde dépend grandement de la maîtrise de la langue maternelle de   l’enfant et un grand nombre de compétences sont interchangeables. Les   questions de l’IMDPE reflètent des compétences et des habiletés   transférables. Donc, si un enfant a maîtrisé ces compétences en anglais, il y   a de très bonnes chances qu’il/elle ait également maîtrisé ces compétences en   français.Nous demandons aux enseignant(e)s d’utiliser leur meilleur jugement   lorsque viendra le temps de répondre aux questions de l’IMDPE reliées aux   compétences qu’ils n’ont pas eu l’occasion d’observer ou lorsque certaines   compétences ou certains comportements ne font pas partie de la philosophie de   la langue française.&lt;/p&gt;&lt;!--&lt;img width=575   src="images/help/image010.gif"alt="Rounded Rectangle: Les   classes d’immersion en anglais font parties d’un programme dans lequel les   élèves de la maternelle sont introduits à un jeune âge à l’anglais. La langue   principale de l’école, cependant, demeure le français. Donc les   enseignant(e)s d’immersion en anglais devraient répondre aux questions de   l’IMDPE en fonction des observations des habiletés des enfants en français.   Il serait donc injuste et erroné de mesurer les habiletés des enfants en   anglais, car la plupart d’entre eux viennent tout juste d’être exposé à la   langue. L’évaluation de leurs habiletés en anglais leur mettrait dans une   situation désavantageuse.&amp;#13;&amp;#10;&amp;#13;&amp;#10;Si   l’enseignant(e) ne se sent pas confortable de répondre à une question en   particulier, il/elle peut sélectionner « ne sais pas   ».&amp;#13;&amp;#10;&amp;#13;&amp;#10;L’acquisition d’une langue seconde   dépend grandement de la maîtrise de la langue maternelle de l’enfant et un   grand nombre de compétences sont interchangeables. Les questions de l’IMDPE reflètent   des compétences et des habiletés transférables. Donc, si un enfant a maîtrisé   ces compétences en anglais, il y a de très bonnes chances qu’il/elle ait   également maîtrisé ces compétences en   français.&amp;#13;&amp;#10;&amp;#13;&amp;#10:Nous demandons aux   enseignant(e)s d’utiliser leur meilleur jugement lorsque viendra le temps de   répondre aux questions de l’IMDPE reliées aux compétences qu’ils n’ont pas eu   l’occasion d’observer ou lorsque certaines compétences ou certains   comportements ne font pas partie de la philosophie de la langue   française.&amp;#13;&amp;#10;"&gt;--&gt;',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,3,1,0,0,'ClassType',0,Null,Null,Null,'Null','Other Immersion','Autre   langue d''immersion','OtherImmersion',1,'LookupSets',1,160,1,1,'NULL'   ,1,'&lt;p&gt;&lt;b&gt;10. Other Immersion&lt;/b&gt;&lt;/p&gt;&lt;p&gt;·   Please indicate if this class is part of an immersion program for a language   other than French.&lt;/p&gt;','&lt;p&gt;&lt;b&gt;10. Autre langue   d’immersion&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Veuillez indiquer si cette classe   fait partie d’un programme d’immersion dans une langue autre que   l’anglais.&lt;/p&gt;',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,7,1,0,0,'ClassType',0,Null,Null,Null,'Null','Child''s First   Language(s)','Langue(s) maternelle(s) de   l''enfant','FirstLanguage1',1,'LookupSets',15,190,1,1,'NULL'   ,0,'','',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,7,1,0,0,'ClassType',0,Null,Null,Null,'Null','','','FirstLanguage2',1,'LookupSets',15,200,1,1,'NULL'   ,0,'','',0,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,7,1,0,0,'ClassType',0,Null,Null,Null,'Null','','','FirstLanguage3',1,'LookupSets',15,210,1,1,'NULL'   ,0,'','',0,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,3,1,0,0,'ClassType',0,Null,Null,Null,'Null','Communicates   adequately in his/her first language','Communique de mani&amp;egravere   ad&amp;eacutequate dans sa langue   maternelle','Communicates',1,'LookupSets',2,220,1,1,'NULL'   ,1,'&lt;p&gt;&lt;b&gt;12. Communicates Adequately in his/her First   Language&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· If the child communicates adequately   in his/her native tongue (based on your observation or parent information)   please indicate &lt;b&gt;Yes&lt;/b&gt;.&lt;/p&gt;&lt;p&gt;· If not please   indicate &lt;b&gt;No&lt;/b&gt;.&lt;/p&gt;&lt;p&gt;· If you are uncertain   please indicate &lt;b&gt;Don’t   Know&lt;/b&gt;.&lt;/p&gt;','&lt;p&gt;&lt;b&gt;12. Communique de manière   adéquate dans sa langue maternelle&lt;/b&gt;&lt;/p&gt;&lt;p&gt;· Si l’enfant   communique de manière adéquate dans sa langue maternelle (selon vos   observations ou selon l’information fournie par les parents), veuillez   indiquer &lt;b&gt;Oui&lt;/b&gt;.&lt;/p&gt;&lt;p&gt;· Sinon, veuillez indiquer   &lt;b&gt;Non&lt;/b&gt;.&lt;/p&gt;&lt;p&gt;· Si vous n’êtes pas certains,   veuillez indiquer &lt;b&gt;Ne Sais   Pas.&lt;/b&gt;&lt;/p&gt;',1,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,14,1,0,0,'ClassType',0,Null,Null,Null,'Null','','','LanguageCompleted',0,'NULL',NULL,350,1,1,'NULL'   ,0,'','',0,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,14,1,0,0,'ClassType',0,Null,Null,Null,'Null','','','CompletedQuestions',0,'NULL',NULL,360,1,1,'NULL'   ,0,'','',0,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,14,1,0,0,'ClassType',0,Null,Null,Null,'Null','','','RequiredQuestions',0,'NULL',NULL,370,1,1,'NULL'   ,0,'','',0,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,14,1,0,0,'ClassType',0,Null,Null,Null,'Null','','','IsComplete',0,'NULL',NULL,380,1,1,'NULL'   ,0,'','',0,0,'null','','')
INSERT INTO   [EDI].[Questionnaires.Configuration]([QuestionnaireID],[InputTypeId],[OrientationID]   ,[IsHeader] ,[ShowGroupName]   ,[GroupName],[IsCollapsible],[min],[max],[numberstep],[Label],[English],[French],[VariableName],[HasLookupEntity],[LookupEntity],[LookupEntityID],[Sequence],[Columns],[ColumnSpan],[QuestionNumber],HasHelp,HelpTextEnglish,HelpTextFrench,IsRequired,HasNotification,NotificationCondition,[Notification],NotificationFrench)   VALUES(11,14,1,0,0,'ClassType',0,Null,Null,Null,'Null','','','QuestionnaireId',0,'NULL',NULL,390,1,1,'NULL'   ,0,'','',0,0,'null','','')






`

**Comment by rennera** [comment]
ServiceContext 
- update questionnaires hasdata - Set ABC to have Ontario
- Insert new questionnaire for Demographics, D and E

**Comment by rennera** [comment]
Update the entities then create the new migration
`dotnet ef migrations add 2021OntarioDemographics --context servicecontext -p ../Infrastructure/Infrastructure.csproj -s Web.csproj -o Data/Migrations`

**Comment by rennera** [comment]
Update the database with the new seeddata(hasdata methods) and entity changes
`dotnet ef database update -c servicecontext`

**Comment by rennera** [comment]
Update the entities for section E then create the new migration
dotnet ef migrations add 2021OntarioSectionE --context servicecontext -p ../Infrastructure/Infrastructure.csproj -s Web.csproj -o Data/Migrations

**Comment by rennera** [comment]
Final Upgrade 
- Backup Prod
- Restore on Dev
- Execute database updates `dotnet ef database update -c servicecontext`
- Execute questionnaire updates `2021-06-11 - Ontario Questionnaire`
- Update Variable names `2021-06-07 - Update Variable Names`
- import ONTARIO students\teachers data

**Comment by rennera** [comment]
Update Release

Build --> PUBLISH Application
	- target framework - 5.0
	- target runtime - Portable
	- Configuration - Release

On Server
1) Open IIS 
2) Open Perfmon (https://hostadvice.com/how-to/how-to-view-the-number-of-active-user-sessions-on-your-iis-website/)
	- Add webservice - Current Anonymous, Current NonAnonymous and Current Connections
	- or check e-edi Active Connections - under User Defined
3) Stop Website in IIS
4) open directory C:\inetpub\wwwroot\e-EDI\publish
5) Archive current publish directory by renaming folder with incremental version number (ie publish v54)
6) Copy Directory from dev machine to server (C:\Users\andre\source\repos\Sources\Offord-EDI-2020\EDI\Web\bin\Release\net5.0\publish)
7) Copy Data Directory from previous publish folder to new folder (ie copy publish v54\Data --> publish\Data)
8) Run [IIS Permissions - After Publish.ps1] postscript file TWO TIMES after copy is complete (this will grant permissions to the upload directories for the e-edi application pool)
9) start IIS 
10) test site
